### DISCUSSION TITLE
Finding Pull requests of others

### DISCUSSION THREAD

**Discussion by pitchedwhite** [microsoft/winget-cli#4087] [General]
Hello i am new to this stuff but i know that with "WingetCreate" you can submit tokens and the thing you are installing with it is deposited into a gigantic repository that others are doing and adding every single day, how can i access this repository of more random installations that do not explicitly appear in "winget search"? I just want to find the ways others have been running installations and packages and can't navigate this site to find it or might even be on the wrong site, please help.

**Comment by mdanish-kh** [comment]
Not sure if I fully understand the question but..

The submissions by WingetCreate (and other manifest creator tools) are made into this repository https://github.com/microsoft/winget-pkgs - The Windows Package Manager Community Repository. It contains all the manifests for packages under the `winget` source. You can view the manifest for each package under the [manifests/](https://github.com/microsoft/winget-pkgs/tree/master/manifests) directory. The directory is partitioned alphabetically so that it's easier to find the package you're looking for.

The following resource may also help you get started:
- https://learn.microsoft.com/en-us/shows/open-at-microsoft/contributing-to-windows-package-manager/
